### DISCUSSION TITLE
Screenshots for F-Droid

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [SagerNet/SagerNet#66] [Q&A] [answered]
You can add Screenshots for F-Droid, too, if you want.
https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/
https://gitlab.com/snippets/1895688
That was simply information about what is still possible. You would then only have to enter the screenshots in fastlane. This is just one way that you can also add screenshots to the app view if you can. If you added screenshots (then F-Droid is happy) and if you can't add any -> then you can close the issue because it was only an info. Thank you that your app is available in
fdroid. Very much for your attention.

it is only information.

**Comment by nekohasekai** [comment, marked as answer]
I have no plans to list F-Droid before it stabilizes, it usually takes days or even weeks to release an update

**Comment by mondlicht-und-sterne** [comment]
its in this repo

https://apt.izzysoft.de/fdroid/repo

https://apt.izzysoft.de/fdroid/index/apk/io.nekohasekai.sagernet

**Comment by nekohasekai** [comment]
No, I don't know it.
